### PR TITLE
clear help msg for ignore_eos to avoid misunderstanding

### PR DESCRIPTION
**Pull request by sywangyi**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Comment by sywangyi** [comment]
@regisss please help review

File: examples/language-modeling/README.md [hunk 1]
```diff
@@ -746,7 +746,7 @@ python3 ../text-generation/run_generation.py \
     --use_kv_cache \
     --batch_size 1 \
     --use_hpu_graphs \
-    --ignore_eos \
+    --no-ignore_eos \
```
Comment (ssarkar2) [inline examples/language-modeling/README.md:749]: Why change `--ignore_eos` to `--no-ignore_eos` only for this cmd?

File: examples/text-generation/run_generation.py [hunk 2]
```diff
@@ -267,7 +267,7 @@ def setup_parser(parser):
         "--ignore_eos",
         default=True,
         action=argparse.BooleanOptionalAction,
-        help="Whether to ignore eos, set False to disable it",
+        help="Whether to disable stopping with eos token when calling `generate`. --no-ignore_eos to disable it",
```
Comment (ssarkar2) [inline examples/text-generation/run_generation.py:270]: **Minor**: For `argparse.BooleanOptionalAction`, if we have `--foo`, then the arg parser automatically provides `--no-foo`. That's mentioned in argparse documentation: https://docs.python.org/3/library/argparse.html ,  so we might not need to explicitly add `--no-ignore_eos to disable it` here.

I think its fine and good to add it, so we can keep it. But maybe we should add it in the help line of `examples/image-to-text/run_pipeline.py` as well. Or we should remove mention of `--no-ignore_eos` from both `examples/text-generation/run_generation.py` and `examples/image-to-text/run_pipeline.py`

**Review by ssarkar2 — COMMENTED** [review]
Thanks for clarifying the help msg. Some minor comments

File: examples/language-modeling/README.md (hunk 1 above)
Comment (sywangyi) [reply examples/language-modeling/README.md:749]: The cmd is shown in Prompt/Prefix/P-tuning section, which uses https://huggingface.co/datasets/ought/raft/viewer/twitter_complaints for finetune, whose label is expected as "complaint" or "no complaint", as a result, the generation output is expected as "complaint" or "no complaint" using the finetuned peft model as input. if not set --no-ignore_eos, extra work will be output.

File: examples/text-generation/run_generation.py (hunk 2 above)
Comment (sywangyi) [reply examples/text-generation/run_generation.py:270]: The `ignore_eos` arg in `examples/text-generation/run_generation.py` does not use `argparse.BooleanOptionalAction`, it's `action="store_true",`, default value is false. and adding `--ignore_eos` in command makes it true. so no need to mention --no-ignore_eos in `examples/text-generation/run_generation.py`

**Comment by libinta** [comment]
@sywangyi  can you fix the review comments?

**Review by ssarkar2 — APPROVED** [review]

**Review by regisss — APPROVED** [review]

**Pull request merged**